src/VatNumberValidator.php

```diff
@@ -90,9 +90,12 @@ public function validate($value, Constraint $constraint)
             return;
         }
 
+        // Use only the expressions for the given country code
+        $schemes = preg_grep($data = '/^\/\^\(' . preg_quote(substr($value, 0, 2), '/') . '/', $this->schemes);
```

Comment:
The `$data` variable is unused, and please make sure the php-cs-fixer check is complete.

Reply:
I removed the `$data`, completely missed that one
Can you point out what you mean with the php-cs-fixer check?
I saw some errors on psr2 coding standard when I run phpcs, but it is not within the lines i wrote. Should I apply the psr2 to the code and fix it for you?

Reply:
In the root of the repo, there is a .php_cs.dist file, which contains the rules I usually apply when formatting my code (https://github.com/FriendsOfPHP/PHP-CS-Fixer). Long story short: please remove the spaces around the string concatenating dots, and than it is ready to be merged. Thank you!

Reply:
Thanks for mentioning the php-cs-fixer. I usually depend on my ide. This helps a lot when dealing with different coding standards.